jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/LoadDockerStep.java

```diff
@@ -94,12 +94,13 @@ private Void afterPushBaseImageLayerFuturesFuture()
     DockerClient dockerClient = new DockerClient();
     dockerClient.load(new ImageToTarballTranslator(image).toTarballBlob(targetImageReference));
 
-    // Tags the image with all the tags. This only needs to be done for additional tags, so this can
-    // be skipped if there is only one tag - the load above already loads that tag.
-    if (buildConfiguration.getAllTargetImageTags().size() > 1) {
-      for (String tag : buildConfiguration.getAllTargetImageTags()) {
-        dockerClient.tag(targetImageReference, targetImageReference.withTag(tag));
+    // Tags the image with all the additional tags, skipping the one 'docker load' already loaded.
+    for (String tag : buildConfiguration.getAllTargetImageTags()) {
+      if (tag.equals(targetImageReference.getTag())) {
```

Comment:
Could invert this logic and remove the continue since only 1 line follows.

Reply:
This is just a personal preference thing, but I preferred to do it this way here because it clearly conveys that the original tag should be skipped (a single special case for the loop), whereas for most of the cases of the loop, the usual logic of performing the tagging should happen. Inverting the logic here conveys the sense of "only do the tagging if the tag is not the original", which is the same logic, but gives a sense that we are filtering for a small subset of the tags that is actually not the original tag.